packages/adapter-utils/src/element_source.ts

```diff
@@ -80,7 +84,8 @@ export const buildElementsSourceFromElements = (
       .find(values.isDefined)
   }
 
-  const has = async (id: ElemID): Promise<boolean> => (await get(id)) !== undefined
+  const has = async (id: ElemID): Promise<boolean> =>
+    getElement(id) !== undefined || awu(fallbackSources).some(async source => source.has(id))
```

Comment:
nit: redundant `async` definition
```suggestion
    getElement(id) !== undefined || awu(fallbackSources).some(source => source.has(id))
```
this actually has an effect on performance in large enough numbers, not the case here, but still...